src/pages/policies-and-schedules/ListContainer.tsx

```diff
@@ -9,43 +9,70 @@ import {
 } from 'react-router-dom';
 import SchedulesList from 'pages/schedules/SchedulesList';
 import PoliciesList from 'pages/policies/PoliciesList';
+import { TabInk } from '../../model';
 
 const items = [
   { label: 'Policies', path: 'policies' },
   { label: 'Schedules', path: 'schedules' },
 ];
 
-const inkStyle = {
-  policies: { width: '105px', left: '0px' },
-  schedules: { width: '122px', left: '105px' },
-};
-
 export const ListContainer = () => {
   const match = useRouteMatch();
   const location = useLocation();
   const pathParts = location.pathname.split('/');
-  const path = pathParts[pathParts.length - 1];
+  const pathEl = pathParts[pathParts.length - 1];
   const pathWithoutTab = location.pathname.split('/').slice(0, -1).join('/');
+  const [inkStyle, setInkStyle] = useState<TabInk[]>([]);
+
+  useEffect(() => {
+    const list = document.querySelectorAll('.tabmenuitem-getwidth');
```

Comment:
Maybe we can do it more simply? for example - bootstrap/primereact has some ready to use components?